src/annotation.js

```diff
@@ -101,7 +101,7 @@ export default {
 
   beforeEvent(chart, args, options) {
     const state = chartStates.get(chart);
-    handleEvent(state, args.event, options);
+    args.changed = handleEvent(state, args.event, options);
```

Comment:
```suggestion
    if (handleEvent(state, args.event, options)) {
        args.changed = true;
    }
```
Should not set the `changed` to false if some other plugin already set it to true before the event was passed to this plugin.